src/com/dotmarketing/business/LoginAsAPI.java

```diff
@@ -0,0 +1,31 @@
+package com.dotmarketing.business;
```

Comment:
The "com/dotmarketing/business" package is where legacy code lives. This new class must be placed inside the new "com/dotcms/api" packages.